src/app/Dashboard/operations/types.js

```diff
@@ -2,8 +2,22 @@ const FETCH_USER_ACTIVITIES_REQUEST = 'FETCH_USER_ACTIVITIES_REQUEST';
 const FETCH_USER_ACTIVITIES_SUCCESS = 'FETCH_USER_ACTIVITIES_SUCCESS';
 const FETCH_USER_ACTIVITIES_ERROR = 'FETCH_USER_ACTIVITIES_ERROR';
 
+const LOG_POINTS = {
```

Comment:
Let's have a common way of declaring constants. There is no need of declaring an object to define your constants. This has been done on the line `17`

Reply:
@Chris-Maina I think declaring constants reduces repetition in certain words. It also makes them organized and still do the same work.

Reply:
@alexmochu  I think @Chris-Maina is saying have a common way of declaring constants e.g on line 1
and there is no need for you to create another object because your action type will look like this `types.LOG_POINTS.POST_SUCCESS`  instead of `LOG_POINTS_SUCCESS`

Reply:
@Kachulio1 @Chris-Maina  Thank you for the feedback. I have reverted to the recommended way of declaring constants